ast.json

```diff
@@ -50,7 +50,7 @@
         "filePath"
       ],
       "docs": {
-        "description": "Get a CSV and convert it to JSON",
+        "description": "Get a CSV and returns a JSON array of strings for each item  separated by the delimiter",
```

Comment:
Either change "get a csv" to "gets a csv" or change "returns a JSON array" to "returns a JSON array".